packaging/os/dnf.py

```diff
@@ -19,18 +20,9 @@
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-
-import traceback
 import os
 import dnf
```

Comment:
dnf library import should catch import exception as there was in the previous version